jenkins/image_building/initrd_sdk/test_unlock_config

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
you don't have executable in this file, no need to add a shebang if you're just sourcing it

Reply:
I would like to have it for the reasons mentioned by @tuminoid .